src/main/java/baseball/service/BaseballService.java

```diff
@@ -0,0 +1,34 @@
+package baseball.service;
+
+import baseball.domain.Baseball;
+import baseball.domain.BaseballStatus;
+
+public class BaseballService {
+    public boolean isStrike(int userNum, int randomNum) {
+        return userNum == randomNum;
+    }
+
+    public boolean isBall(int userNum, int userIdx, Baseball randomBall) {
+        if (!isStrike(userNum, randomBall.getBaseballs().get(userIdx))) {
+            return randomBall.getBaseballs().contains(userNum);
+        }
+        return false;
+    }
+
+    public BaseballStatus compare(Baseball userBall, Baseball randomBall) {
+        int strike = 0, ball = 0;
```

Comment:
하나의 선언문에는 하나의 변수만 생성하는 것이 좋습니다. 아래 [코딩 컨벤션](https://naver.github.io/hackday-conventions-java/) 을 참고하시면 좋을 것 같아요!
<img width="1015" alt="스크린샷 2022-05-28 오후 12 28 29" src="https://user-images.githubusercontent.com/92219795/170808134-f08a8a85-3857-4317-9bf8-b28c2e76f239.png">

```suggestion
        int strike = 0;
        int ball = 0;
```